source/chapters/configuration.rst

```diff
@@ -160,31 +160,32 @@ computer. **Therefore it is important to take care to choose the best Audio API
 available to you.** Refer to the following table of Audio APIs to see what the
 best choice is for your operating system.
 
-+----------------------------------------+--------------+
-| OS / Audio API                         | Quality      |
-+========================================+==============+
-| Windows / WMME                         | Poor         |
-+----------------------------------------+--------------+
-| Windows / DirectSound                  | Poor         |
-+----------------------------------------+--------------+
-| Windows / WASAPI                       | Good         |
-+----------------------------------------+--------------+
-| Windows / ASIO                         | Good         |
-+----------------------------------------+--------------+
-| Windows / WDDKMS                       | Good         |
-+----------------------------------------+--------------+
-| Mac OS X / CoreAudio                   | Good         |
-+----------------------------------------+--------------+
-| GNU Linux / OSS                        | OK           |
-+----------------------------------------+--------------+
-| GNU Linux / ALSA                       | Good         |
-+----------------------------------------+--------------+
-| GNU Linux / JACK (Advanced)            | Good         |
-+----------------------------------------+--------------+
-
-On Windows, if an ASIO driver is not available for your operating system, you
-can try installing `ASIO4ALL <http://asio4all.com>`_, a low-latency audio driver
-for WDM audio devices.
++-----------------------------+---------+
+| OS / Audio API              | Quality |
++=============================+=========+
+| Windows / ASIO              | Good    |
++-----------------------------+---------+
+| Windows / WDM-KS            | Good    |
++-----------------------------+---------+
+| Windows / WASAPI            | OK      |
++-----------------------------+---------+
+| Windows / DirectSound       | Poor    |
++-----------------------------+---------+
+| Windows / MME               | Poor    |
++-----------------------------+---------+
+| Mac OS X / CoreAudio        | Good    |
++-----------------------------+---------+
+| GNU Linux / ALSA            | Good    |
++-----------------------------+---------+
+| GNU Linux / JACK (Advanced) | Good    |
++-----------------------------+---------+
+| GNU Linux / OSS             | OK      |
++-----------------------------+---------+
+
+On Windows, if all your soundcards comes with an ASIO driver bypassing the 
+kernel, it is the best choice to achieve a low latency. If this is not the 
+case, use the WDM-KS API. There is generally no advantage 
+to using ASIO4ALL <http://asio4all.com>_, a wrapper around the WDM-KS API. 
```

Comment:
Better, but I think there should be a little explanation of why bypassing the kernel mixer matters.
"On Windows, if your sound card has an ASIO driver, it is best to use that. Using ASIO, Mixxx can run with low latency because ASIO bypasses the sound processing that the Windows kernel mixer usually does. If there is no ASIO driver for your sound card, use the WDM-KS API."